db/migrate/20220714183808_create_pg_search_documents.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePgSearchDocuments < ActiveRecord::Migration[6.0]
```

Comment:
Das sollten wir nicht in diesem Gem machen, da es sonst sein kann, dass wir über die Zeit mit pg_search divergieren. Statt dessen sollten wir in der README erwähnen den Generator von pg_search auszuführen. 
https://github.com/Casecommons/pg_search#setup

Reply:
Bin ich mir nicht sicher, da die Tabelle auch bearbeit wird und die page_id mit integriert wird. Wenn sich die Migrationen anpassen, müsste das Gem eh angepasst werden.

Reply:
Ja, das ist in der Tat wichtig. Wir sollten trotzdem die beiden Migrationen trennen (also die PgSearch Migration aus dem Gem kommen lassen). Und in unserem Gem die `page_id` in den Table hinzufügen. Die Trennung ist wichtig im Kontext einer Nutzung in einer Rails app, die wir nicht kennen.